### PR TITLE
Text added, describing the usage of media specific attributes.

### PR DESCRIPTION
**Pull request by cdh4u**

### CODE REVIEW

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml [hunk 1]
```diff
@@ -386,7 +386,17 @@
...
         associating SDP attributes with bundled "m=" lines.
-      </t>
+        </t>
+        <t>
+        The semantics of some attributes only apply to specific types of media.
+        For example, the semantic of the SDP 'rtcp-mux' and 'rtcp-mux-only' attributes
+        only apply to "m=" lines describing RTP-based media. However, as described in
+        <xref target="sec-sdp-oa-cat" />, IDENTICAL and TRANSPORT mux category SDP attributes
+        <xref target="I-D.ietf-mmusic-sdp-mux-attributes" /> are only
+        associated with the "m=" line associated with the BUNDLE-tag, it means that
```
Comment (taylor-b) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Just a grammatical thing: "which means" instead of "it means"

Comment (martinthomson) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: or ". That means"

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml [hunk 2]
```diff
@@ -386,7 +386,17 @@
...
+        media specific attributes might be associated with an "m=" line describing
```
Comment (martinthomson) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: s/might/can

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml (hunk 1 above)
Comment (cdh4u) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: I can do "that means".

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml (hunk 2 above)
Comment (cdh4u) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Will fix.

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml [hunk 3]
```diff
@@ -438,11 +431,13 @@
...
-          When an offerer associates SDP attributes with a bundled "m=" line associated
-          with a shared address, IDENTICAL and TRANSPORT mux category SDP attributes
-          <xref target="I-D.ietf-mmusic-sdp-mux-attributes" /> are associated with the
-          "m=" line only if the "m=" line is also associated with the offerer BUNDLE-tag.
-          Otherwise the offerer MUST NOT associate such SDP attributes with the "m=" line.
+          When an offerer associates SDP attributes with a bundled "m=" line (including any
+          bundle-only "m=" line) associated with a shared address, IDENTICAL and TRANSPORT
+          mux category SDP attributes <xref target="I-D.ietf-mmusic-sdp-mux-attributes" />
+          are associated with the "m=" line only if the "m=" line is also associated with
+          the offerer BUNDLE-tag. Otherwise the offerer MUST NOT associated such SDP attributes
```
Comment (martinthomson) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: associate

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml [hunk 4]
```diff
@@ -451,10 +446,19 @@
...
+          and attribute values, there is no need to associate such SDP attributes with each "m=" line.
+          The attributes and attribute values are implicitly applied to each "m=" line.
+        </t>
+        <t>
+          The semantics of some SDP attributes only apply to specific types of media.
+          For example, the semantics of the SDP 'rtcp-mux' and SDP 'rtcp-mux-only' attributes
+          only apply to "m=" lines describing RTP-based media. However, as described in
+          <xref target="sec-sdp-oa-cat" />, there are cases where IDENTICAL and TRANSPORT
+          mux category SDP attributes are only associated with the "m=" line associated with
+          the BUNDLE-tag. That means that media specific IDENTICAL and TRANSPORT mux category
```
Comment (martinthomson) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: media-specific

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml [hunk 5]
```diff
@@ -1397,85 +1399,95 @@ SDP Answer
...
-            When an offerer generates an initial offer, the
-            offerer MUST associate an SDP 'rtcp-mux' attribute
-            <xref format="default" pageno="false" target="RFC5761"/>
-            with each bundled RTP-based "m=" line in the offer, including
-            a bundle-only "m=" line. In addition, the offerer MUST
-            associate an SDP 'rtcp-mux-only' attribute
-            <xref format="default" pageno="false" target="I-D.ietf-mmusic-mux-exclusive"/>
-            with each RTP-based bundle-only "m=" line, and MAY associated an SDP
-            'rtcp-mux-only' attribute with other bundled RTP-based "m=" lines.
+            When an offerer generates an initial offer, if the offer contain
```
Comment (martinthomson) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: contains

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml [hunk 6]
```diff
@@ -1397,85 +1399,95 @@ SDP Answer
...
+            one or more RTP-based bundled "m=" lines (or, if there is a chance that RTP-based
+            "m=" lines will later be added to the BUNDLE group), the offerer MUST
+            associated an SDP 'rtcp-mux' attribute <xref format="default"
```
Comment (martinthomson) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: associate

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml (hunk 3 above)
Comment (cdh4u) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Fixed.

I will also combine the offerer- and answerer paragraphs into a single paragraph, in oder to get rid of some of the repetition you mentioned.

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml (hunk 4 above)
Comment (cdh4u) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Fixed.

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml (hunk 5 above)
Comment (cdh4u) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Fixed.

File: draft-ietf-mmusic-sdp-bundle-negotiation.xml (hunk 6 above)
Comment (cdh4u) [inline draft-ietf-mmusic-sdp-bundle-negotiation.xml]: Fixed.

**Pull request merged**